apps/raptor/lib/raptor_web/router.ex

```diff
@@ -8,5 +8,6 @@ defmodule RaptorWeb.Router do
   scope "/api", RaptorWeb do
     pipe_through :api
     get("/authorize", AuthorizeController, :authorize)
+    get("listAccessGroups", ListAccessGroupsController, :list)
```

Comment:
RaptorWeb still confuses me a bit

What utilizes listAccessGroups? It's not a normal route like `/authorize`? (`listAccessGroups` vs `/listAccessGroups`)